<launch>

  <!-- Data generation node -->
  <node pkg="data_generation" type="gt_generation_node" name="gt_generation" output="screen">
    <rosparam command="load" subst_value="true" file="$(find data_generation)/config/gt_generation_node.yaml" />
  </node>

  <!-- Rviz -->
  <node pkg="rviz" type="rviz" name="$(anon rviz)" output='log'
    args="-d $(find data_generation)/launch/rviz/gt_generation.rviz" />

</launch>